Source/WebCore/platform/graphics/avfoundation/objc/MediaPlayerPrivateMediaSourceAVFObjC.mm

```diff
@@ -356,14 +356,12 @@ void getSupportedTypes(HashSet<String, ASCIICaseInsensitiveHash>& types) const f
         return;
 
 #if HAVE(AVSAMPLEBUFFERRENDERSYNCHRONIZER_RATEATHOSTTIME)
-ALLOW_NEW_API_WITHOUT_GUARDS_BEGIN
-    if (hostTime && [m_synchronizer respondsToSelector:@selector(setRate:time:atHostTime:)]) {
```

Comment:
This method was only added in macOS 11.3, I believe WebKit still needs to build on macOS 11 for a little longer – but I might be wrong.

Reply:
No it does not anymore. Now we only support 12 and up.

Reply:
You're right.